src/style.css

```diff
@@ -20,7 +23,92 @@ body {
   justify-content: space-between;
 }
 
+.header {
+  font-size: 70px;
+  margin-top: 0;
+  margin-left: 10%;
+  color: #005638;
+}
+
+.container {
+  margin-left: 10%;
+}
+
+.recent-scores {
+  justify-content: space-between;
+  align-items: baseline;
+  padding-right: 5px;
+}
+
+.recent-scores p {
+  flex-basis: 60%;
+  text-align: center;
+  font-size: 50px;
+  margin-top: 0;
+  color: #005638;
+}
+
+.score-container {
+  flex-basis: 40%;
+}
+
+.bg-gradient {
+  border-radius: 10px;
+  background: linear-gradient(#36d1dc, #5b86e5);
+}
+
+.best-scores {
+  border-radius: 10px;
+  margin: 5px;
+  padding: 3%;
+  background: linear-gradient(#a8ff78, #78ffd6);
+}
+
 .top-scores {
   justify-content: space-between;
   list-style: none;
 }
+
+.top-scores p {
```

Comment:
[OPTIONAL] I think it could be nice if either you validate that names from users are not too long or you can truncate the text after a certain length. Which one do you like the most?

![image](https://user-images.githubusercontent.com/79114233/141418867-369f5e88-ce8a-4ee5-9161-a9f397887fd1.png)

Reply:
Hi! @D4nielJ hope this message finds you well!
Thank you for your suggestion I already worked on it and even I added other functionalities for UX.
Feel free to take a look and if you like it I will appreciate a star to my repo 😄 ⭐ ✅ 
Cheers!
Alex!